hydra-cluster/exe/hydra-cluster/Main.hs

```diff
@@ -23,17 +29,23 @@ run options =
   withTracer (Verbose "hydra-cluster") $ \tracer -> do
     traceWith tracer ClusterOptions{options}
     let fromCardanoNode = contramap FromCardanoNode tracer
+    let blockfrostNetworks = [BlockfrostPreview]
```

Comment:
why do we only support blockfrost-preview?

Reply:
That's the only key we have from blockfrost people for now.